config/app.config

```diff
@@ -53,6 +53,7 @@ irsa.hips.masterUrl="https://irsa.ipac.caltech.edu/data/hips/list"
 
 wise.ibe.host       = "irsa.ipac.caltech.edu/ibe"
 twomass.ibe.host    = "irsa.ipac.caltech.edu/ibe"
+ztf.ibe.host        = "irsadev.ipac.caltech.edu/ibe"
```

Comment:
This should be `irsa` and not `irsadev`.  `irsadev` can be used at the `dev` env block if needed.